hazelcast/src/main/java/com/hazelcast/scheduledexecutor/impl/ScheduledTaskDescriptor.java

```diff
@@ -38,11 +44,24 @@
     private ScheduledFuture<?> scheduledFuture;
 
     /**
-     * Stats are written from a single thread and read by many outside the partition threads. (see. Member owned tasks)
+     * Stats are written from a single thread and read by many outside the partition threads.
```

Comment:
I consider the Atomic* API to be cleaner and easier to notice during refactoring, and since it comes _almost_ cost-free (ignoring the ref indirection :) ) I would prefer to keep it this way, unless you have a strong argument against it.